src/parser.rs

```diff
@@ -264,6 +265,13 @@ impl Library {
     ) -> Result<(), String> {
         if let Some(typ) = self.read_record(parser, ns_id, elem, None, None)? {
             let name = typ.get_name();
+            // TODO: Don't hardcode to "Vulkan"
+            let typ = if self.namespace(ns_id).name.eq("Vulkan") {
+                assert!(typ.is_incomplete(self));
```

Comment:
We probably won't keep this assert but it shows that Vulkan record are incomplete.
Unfortunately, non Vulkan record can be incomplete too so we can't use `is_incomplete` as trigger for the new behavior.

And then there is the problem of `ash::vk::{}` format string: where should it be specified if we want to externalize it (to make `gir` Vulkan agnostic again).

Reply:
Likewise, instead of tweaking the Vulkan namespace on the fly, we could introduce a postprocess hook that tweaks the namespace after it was parsed.